source/style-guide/style/capitalization.txt

```diff
@@ -88,6 +88,9 @@ Following are some examples.
    * - API
      - application programming interface
 
+   * - fCV
+     - Feature Compatibility Version
```

Comment:
[thought, no change required]

I wasn't sure whether we'd want to use title case for the spelled-out term, but it looks like we use title case more often in the server docs instead of all lower.

Reply:
We're definitely bucking the trend (74 -29), but it would be better lower case. I'll update the docs accordingly

Done